code/modules/mining/mine_turfs.dm

```diff
@@ -76,14 +76,6 @@ var/global/list/rockTurfEdgeCache = list(
 /turf/simulated/mineral/proc/Spread(var/turf/T)
 	new src.type(T)
 
-/hook/startup/proc/add_mineral_edges()
-	var/watch = start_watch()
-	log_startup_progress("Reticulating splines...")
-	for(var/turf/simulated/mineral/M in GLOB.mineral_turfs)
-		M.add_edges()
-	log_startup_progress(" Splines reticulated in [stop_watch(watch)]s.")
```

Comment:
Not allowed to get rid of Reticulating splines, sorry 😛 

This has to exist somewhere.

Reply:
Ill tack this onto the init of Icon smoothing

Reply:
See my other comment